src/components/form/form.stories.mdx

```diff
@@ -234,7 +223,7 @@ It is possible to render `Form` as a content of `Dialog` component.
   </Story>
 </Preview>
 
-With sticky footer
+#### With sticky footer
```

Comment:
why we are using here `####`?

Reply:
Because it is still part of the wider "In Dialog" example. If we wanted to make it ### Then I think the title should be changed to "In dialog with sticky footer"